README.md

```diff
@@ -0,0 +1,145 @@
+<p align="center">
+  <a href="https://pennylane.ai">
+    <img width=80% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/pennylane_thin.png">
+  </a>
+</p>
+
+<p align="center">
+  <!-- Tests (GitHub actions) -->
+  <a href="https://github.com/PennyLaneAI/pennylane/actions?query=workflow%3ATests">
+    <img src="https://img.shields.io/github/workflow/status/PennyLaneAI/pennylane/Tests/master?logo=github&style=flat-square" />
+  </a>
+  <!-- CodeCov -->
+  <a href="https://codecov.io/gh/PennyLaneAI/pennylane">
+    <img src="https://img.shields.io/codecov/c/github/PennyLaneAI/pennylane/master.svg?logo=codecov&style=flat-square" />
+  </a>
+  <!-- ReadTheDocs -->
+  <a href="https://pennylane.readthedocs.io">
+    <img src="https://img.shields.io/readthedocs/pennylane.svg?logo=read-the-docs&style=flat-square" />
+  </a>
+  <!-- PyPI -->
+  <a href="https://pypi.org/project/PennyLane">
+    <img src="https://img.shields.io/pypi/v/PennyLane.svg?style=flat-square" />
+  </a>
+  <!-- Forum -->
+  <a href="https://discuss.pennylane.ai">
+    <img src="https://img.shields.io/discourse/https/discuss.pennylane.ai/posts.svg?logo=discourse&style=flat-square" />
+  </a>

```

Comment:
Do we want to include the python version badge?

Reply:
I removed it since it was starting to cause a line overflow! And I figured it was the most 'superfluous' of all the badges.

Reply:
Ok, sounds good!